### PR TITLE
Client certificates for consumed REST services

### PR DESCRIPTION
**Pull request by joostverhoog**
In Mendix 7.2 we have added a new feature in which you can use client certificates when consuming REST services.

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
Waiting for review.

File: howto7/integration/use-a-client-certificate.md [hunk 1]
```diff
@@ -27,21 +27,21 @@ To configure the custom settings that are only used when you run your app locall
...
     ClientCertificatePasswords | The password for each certificate file.  | In the same order as the **ClientCertificates**.
-    WebServiceClientCertificates **(Optional)** | The description of which files to use for which service, in the format of `"ModuleName.WebserviceName": "full path"`. | If you have more than one web service to configure, you can separate them with commas. The whole setting value needs to be enclosed in curly brackets ["{ }"]. Backslashes in the paths must be doubled. In addition, the paths you specify here should all appear in **ClientCertificates**.
+    ClientCertificateUsages **(Optional)** | The description of which files to use for which service, in the format `"ModuleName.WebserviceName": "full path"` (for web services) or `"www.server-to-contact.com": "full path"` (for REST services). | If you have more than one service to configure, you can separate them with commas. Enclose the whole setting value in curly brackets `{ }`. Backslashes in the paths must be doubled. In addition, the paths you specify here should all appear in **ClientCertificates**.
```
Comment (kevindullemond) [inline howto7/integration/use-a-client-certificate.md]: Please change both occurrences of full path to path

Comment (joostverhoog) [inline howto7/integration/use-a-client-certificate.md]: Ok, done

**Comment by kevindullemond** [comment]
I reviewed it and agree to the changes.

**Pull request merged**